### PR TITLE
feat(Media): add new AudioSession class for configuring audio on iOS

### PR DESCRIPTION
**Pull request by vadymmarkov**
- Add `AudioSession` and `AudioConfiguration`
- Configure audio session in `WebRTCMediaConnection`
- Add audio route picker view in demo

### CODE REVIEW

File: Examples/Conference/App/Shared/Screens/Alias/AliasViewModel.swift [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 import SwiftUI
 import PexipInfinityClient
+import AVFoundation
```
Comment (drymarau) [inline Examples/Conference/App/Shared/Screens/Alias/AliasViewModel.swift:18]: Seems to be the only change in the file, is it necessary?

**Review by drymarau — APPROVED** [review]
Looks good, seems that opening the picker pauses the video feed?

**Comment by vadymmarkov** [comment]
Looks like `AVRoutePickerView` stops camera, which is a bit unfortunate.

**Comment by drymarau** [comment]
> Looks like `AVRoutePickerView` stops camera, which is a bit unfortunate.

No biggie.

**Pull request merged**